File: Python3/Secuenciales/ej2/ej2.py
# -*- coding: utf-8 -*-
#Calcular el perímetro y área de un rectángulo dada su base y su altura.
#1.pedir base
#2.almacenarla en variable
#3.pedir altura
#4.almacenarla en variable
#5.realizar calculo
#6.mostrar resultado

print("==Calcular área y Perímetro de un rectángulo==")
base = float(input("Introduce el valor de la base> "))
altura = float(input("Introduce el valor de la altura> "))
print("El valor del perímetro es "+str((base+altura)*2))
print("El valor del área es "+str(base*altura))

File: Java/Secuenciales/ej2.java
import java.util.Scanner; //Importar la clase Scanner 
/**
 * Calcular el perímetro y área de un rectángulo dada su base y su altura.
 * @author Alberto Miguel Martinez Jimenez
 */

//Almacenar base
//Almacenar altura
//calcular perimetro = (base+altura)*2
//calcular area = base*altura
//mostrar perimetro
//mostrar area

public class ej2 {

	public static void main(String[] args) {
		double area;
		double perimetro;
		double base;
		double altura;
		
		Scanner scanner = new Scanner(System.in);//Intanciar el objeto Scanner	
        
		System.out.print("Introduce la base del rectangulo> ");
		base = scanner.nextDouble();
		
		System.out.print("Introduce la altura del rectangulo> ");
		altura = scanner.nextDouble();
		
		area = base*altura;
		perimetro = (base+altura)*2;
		
		System.out.print("El rectangulo de base "+base+" y altura "+altura+" tiene un area de "+area+" y un perimetro de "+perimetro);
	}

}
